pynamodb/expressions/__init__.py

```diff
@@ -0,0 +1,35 @@
+import re
```

Comment:
ya that looks nice, but there's a fun behavior where people do something like
```python
import pynamodb.expressions
# stuff
o = pynamodb.expressions.condition.Operand(attr)
get_projection_expression() # this resolves 
```
it's not a big deal, and again, python people probably have better opinions than i do on this, but i always have trouble finding function definitions when they're defined in the `__init__` file.

as for the api perspective, we could just put these functions into a class that has the `placeholders` defined on it possibly. i'm generally uncomfortable mutating parameters passed into funcs (`placeholders`) 

the problem with that then of course being that the model + connection/base need reference to the same placeholder instance